cli.js

```diff
@@ -52,7 +52,13 @@ const cli = meow(`
     },
     port: {
       type: 'string',
-      alias: 'p'
+      alias: 'p',
+      default: '8080'
```

Comment:
needed to supply these because i think `webpack-serve` was previously providing this default